Calculator.rb

```diff
@@ -0,0 +1,24 @@
+puts 'What calculation would you like to do? (add, sub, mult, div)'
```

Comment:
This works, but let's try creating a ```Class Calculator``` that has its own methods for multiply, divide, and so on:
```
Class Calculator
   def initialize
   end

end
```